recipes/bwamem2/meta.yaml

```diff
@@ -0,0 +1,26 @@
+{% set version = "2.0" %}
+package:
+  name: bwamem2
+  version: {{ version }}
+
+source:
+  url: https://github.com/bwa-mem2/bwa-mem2/releases/download/v2.0/bwa-mem2-{{ version }}_x64-linux.tar.bz2
+  sha256: 15df322cdb6ff3d12105ee1e263449a9dd7698617b641167bca575d93361489f
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
your binary will only work on Linux, isn't it? So you can not use generic and probably need to skip OSX.

Better would be to compile for OSX or use a special OSX binary if available.